JuiceMaker/JuiceMaker/Model/JuiceMaker.swift

```diff
@@ -1,12 +1,66 @@
-//
-//  JuiceMaker - JuiceMaker.swift
-//  Created by yagom. 
-//  Copyright © yagom academy. All rights reserved.
-// 
-
 import Foundation
 
-// 쥬스 메이커 타입 
 struct JuiceMaker {
     
+    var strawberry = Strawberry()
+    var banana = Banana()
+    var kiwi = Kiwi()
+    var pineapple = Pineapple()
+    var mango = Mango()
+    
+    enum JuiceMakerError: Error {
+        case countUnderZero
```

Comment:
에러타입 정의 너무 잘해주셨어요!
하지만 `재고가 0미만이야` 보다 이 에러를 쓰는 사용자입장에서 `재고부족` 이 좀 더 직관적이지 않을까요? ㅎㅎ

```suggestion
        case insufficientFruit
```
```suggestion
        case outOfStock
```

[+]
어떤 과일의 재고가 얼마나 부족한지 alert에 띄워주려면 case를 어떻게 변경하면 좋을까요?